pkg/router/controller/status_test.go

```diff
@@ -735,7 +735,7 @@ func TestFindOrCreateIngress(t *testing.T) {
 	}
 
 	routerName := "foo"
-	ingress, changed := findOrCreateIngress(route, routerName)
+	ingress, changed := findOrCreateIngress(route, routerName, "")
 	if !changed {
 		t.Errorf("expected the route list to be changed: %#v", route.Status.Ingress)
 	}
```

Comment:
@smarterclayton spoke with @knobunc about this. This will be done as a follow on. We are trying to make the 3.5 deadline today.